tests/foreman/api/test_convert2rhel.py

```diff
@@ -155,6 +161,9 @@ def centos(
     ]
     act_key = create_activation_key(target_sat, module_org, module_lce, cv, c2r_sub.id)
     register_host(target_sat, act_key, module_org, smart_proxy_location, centos_host)
+    centos_host.execute("yum update kernel* -y")
+    if centos_host.execute('needs-restarting -r').status == 1:
+        centos_host.power_control(state='reboot', ensure=True)
```

Comment:
ensure=True is default,  so can be removed
```suggestion
        centos_host.power_control(state='reboot')
```

Reply:
Updated.